branchinfo.txt

```diff
@@ -2,10 +2,10 @@
 # Any line that is not blank and does not start with '#' is interpreted as a variable to set.
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=2
-MINOR_VERSION=0
+MINOR_VERSION=1
```

Comment:
@chcosta - we should look at what is still using the branchinfo.txt file and switch it over to use the dir.props file.

I'm fairly certain this file should be deleted by now.